package.json

```diff
@@ -13,16 +11,20 @@
     "node": ">=16.18.0"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
-    "test": "tsdx test",
-    "test:watch": "tsdx test --watch",
-    "test:e2e": "tsdx test --config jest-e2e-config.json",
+    "start": "nodemon -e ts,tsx,json,js,jsx --watch ./src --ignore ./dist --exec 'yarn build --incremental'",
```

Comment:
typescript seems to support this natively; is nodemon doing something that tsc doesn'tdo (or doesn't do _well_) ?

https://www.typescriptlang.org/docs/handbook/configuring-watch.html

Reply:
Normally this would be acceptable but we'd need to repeat multiple commands to build all with `--watch` whereas `nodemon` sits outside of the compiler and can issue arbitrary commands on file change.

Reply:
Ah, so this means that we'll be using it in more places than just `yarn start` in future?

Reply:
Yep!